projects/kubernetes/autoscaler/HELM_GIT_TAG

```diff
@@ -0,0 +1 @@
+cluster-autoscaler-chart-9.21.0
```

Comment:
Is this where we need move this to the release branch and handle this as related branched charts like images?